js/app/application.js

```diff
@@ -43,6 +43,19 @@ const KnowledgeSearchIface = '\
   </interface> \
 </node>';
 
+const GrandCentralContentIface = '\
+<node> \
+  <interface name="com.endlessm.GrandCentralContent"> \
+    <method name="ContentQuery"> \
```

Comment:
Yeah I think that's a fair point. I'll give it a more specific name and then if we want to add more functionality we can do so without necessarily introducing API breaks.